src/utils/arrayUtils.ts

```diff
@@ -25,3 +25,17 @@ export async function callChunked<Item>(
     await processFn(arrayChunk)
   }
 }
+
+/**
+ * Return a copy of the given array without null or undefined values
+ */
+export function removeNullish<T>(array: readonly (T | null | undefined)[]): T[] {
```

Comment:
nifty TS trick - by using `<const T>` here you can narrow down returned types to exactly the values that are passed, if known! (e. g. you'll get 'a' | 'b' instead of string, if you pass ['a', 'b']).

See https://github.com/kibertoad/validation-utils/blob/c80f4bbba4d15a8702399bf3ed5d3ae19130d0fb/lib/validation.helper.ts#L140 for an example

Reply:
You are completely right!! I forgot about it, thank you so much Igor 🙇